tests/unit/test-cmd/test.run.js

```diff
@@ -1,6 +1,6 @@
 /* @flow */
 import path from 'path';
-import {EventEmitter} from 'events';
+import EventEmitter from 'events';
```

Comment:
:+1:

I was wondering how was it working, and it looks like we were using the 0.10 compat named export: https://github.com/nodejs/node/blob/master/lib/events.js#L14-L17